charts/agent/README.md

```diff
@@ -173,6 +173,7 @@ The following table lists the configurable parameters of the Sysdig chart and th
 | `extraVolumes.volumes`                                  | Specifies the additional volumes to mount in the sysdig agent to pass new secrets or configmaps                                                                                          | `[]`                                                                                                      |
 | `extraVolumes.mounts`                                   | Specifies the mount points for additional volumes                                                                                                                                        | `[]`                                                                                                      |
 | `extraSecrets`                                          | Allows passing extra secrets that can be mounted via extraVolumes                                                                                                                        | `[]`                                                                                                      |
+| `customSecurityContext`                                 | Allows overriding the securityContext so to overcome limitations and bugs of certain OSes. **WARNING** Only set this as the last resort and only for very well-known issues              | `[]`                                                                                                      |
```

Comment:
I would personally try to make this sound a little less scary (even though you're right to discourage its use). What about this?

```
Allows overriding the Security Context of the Agent pod(s). Primarily intended resolving targeted incidents and debugging.
```

Reply:
ok